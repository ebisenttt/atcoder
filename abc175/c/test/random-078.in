-365832243119737 296622328928162 198929267733135
