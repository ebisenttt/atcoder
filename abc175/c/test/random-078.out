32026292346533
